tests/conftest.py

```diff
@@ -89,14 +91,21 @@ def bot_info():
     return get_bot()
 
 
+class DictRequest(Request):  # Used for monkey patching attributes
```

Comment:
Maybe just add a comment in front of all 3 classes explaning that they're here for monkey patching due to slots and `__dict__`